solver/progress.go

```diff
@@ -43,9 +43,13 @@ const (
 	LogOutputPlain
 )
 
-func WithLogOutput(con Console, logOutput LogOutput) ProgressOption {
+func WithLogOutput(stderr io.Writer, logOutput LogOutput) ProgressOption {
```

Comment:
Why pass in an `io.Writer` type if it's expected to satisfy the `solver.Console` interface? Is this something that will be relaxed in the future?

Reply:
If you only use plain log output, you don't need a `console.Console`. It's only because of how the `NewPrinter` function is written in buildx. We can either contribute to buildx or create an adjusted copy of the function.

Reply:
https://github.com/docker/buildx/pull/937

Reply:
Updated to use new buildx printer.